lib/wibox/widget/imagebox.lua

```diff
@@ -1,6 +1,16 @@
 ---------------------------------------------------------------------------
+-- A widget to render image(s).
+--
+-- The `wibox.widget.imagebox` is part of the Awesome WM's wiboxes system.
+-- It renders image in a box widget.
```

Comment:
Should this be `images`?
I am not sure if the `box`-part of the sentence adds much meaning. I guess it is there just to explain the name `imagebox`?
How about `This widget displays an image.`?

(Feel free to just ignore this.

Reply:
Yeah... Summary/Description still need to be improved. I need to come back on this later. I lake inspiration for now...

> I am not sure if the `box`-part of the sentence adds much meaning. I guess it is there just to explain the name `imagebox`?

You're right. I tried to use the work "box" because it's in the name. Obviously it was a bad idea.

> How about `This widget displays an image.`?

Simple, easy, clear. I actually like it! However, I have a preference for the "render" word because you can resize/use a cairo surface for the image. So render seems more appropriate to me.